packages/types/custom/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # CHANGELOG
 
+## unpublished
+
+- Feature: Add parameter `renderType` to `GeoLocationConfiguration`.
+- Feature: Add type for `LegendConfiguration` and add it to `MapConfig`.
```

Comment:
Can't seem to see the change in `packages/types/custom/core.ts` 😅

Reply:
Oh right, I've removed that again. Good catch. https://github.com/Dataport/polar/pull/21/commits/396325232ee40f29bcef566059879e6b0a57e788